src/FsAutoComplete.BackgroundServices/Program.fs

```diff
@@ -105,7 +105,7 @@ type FsacClient(sendServerRequest: ClientNotificationSender) =
 type BackgroundServiceServer(state: State, client: FsacClient) =
     inherit LspServer()
 
-    let checker = FSharpChecker.Create(projectCacheSize = 1, keepAllBackgroundResolutions = false, suggestNamesForErrors = true)
+    let checker = FSharpChecker.Create(projectCacheSize = 1, keepAllBackgroundResolutions = false, suggestNamesForErrors = false)
```

Comment:
We make `projectCacheSize` 200 by default, it should probably just be 1000 or something silly. I highly recommend increasing its size. It significantly increases the CPU and memory usage of FCS for multi-project solutions because of a high degree of cache thrashing. Whatever guidance existed about this number before we did an actual investigation of this is wrong. See here for more info: https://github.com/dotnet/fsharp/issues/10217

Reply:
Oh, this is in the background service. Hmm. Not sure what to think of that, I assum eit only touches one project